shub/settings/config.py

```diff
@@ -159,3 +179,89 @@
     #    'globus',
     #    'saml_auth'
 ]
+
+# Any plugins enabled from the environment?
+PLUGINS_ENABLED += get_sregistry_envar_list("SREGISTRY_PLUGINS_ENABLED")
+
+## Environment
+
+# Custom additions are handled before this
+# Limit defaults that user can set
+BOOLEAN_DEFAULTS = [
+    "ENABLE_GOOGLE_AUTH",
+    "ENABLE_TWITTER_AUTH",
+    "ENABLE_GITHUB_AUTH",
+    "ENABLE_GITLAB_AUTH",
+    "ENABLE_BITBUCKET_AUTH",
+    "ENABLE_GITHUB_ENTERPRISE_AUTH",
+    "USER_COLLECTIONS",
+    "PRIVATE_ONLY",
+    "DEFAULT_PRIVATE",
+    "DISABLE_BUILDING",
+    "SREGISTRY_GOOGLE_BUILD_SINGULARITY_VERSION",
```

Comment:
A bit unfortunate that this will have 2x `SREGISTRY_` but I can't think of an easy way around it without renaming and that would break backwards compatibility.

Reply:
Also this variable is a string and not a boolean.